service/src/test/java/bio/terra/cbas/controllers/VerifyPactsAllControllers.java

```diff
@@ -45,13 +46,18 @@
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.web.servlet.MockMvc;
 
 @WebMvcTest
-@ContextConfiguration(classes = {RunSetsApiController.class, CbasApiConfiguration.class})
+@ContextConfiguration(
+    classes = {RunSetsApiController.class, MethodsApiController.class, CbasApiConfiguration.class})
+@TestPropertySource(properties = "cbas.cbas-api.runSetsMaximumRecordIds=100")
+@TestPropertySource(properties = "cbas.cbas-api.maxWorkflowInputs=100")
+@TestPropertySource(properties = "cbas.cbas-api.maxWorkflowOutputs=40")
 @Provider("cbas")
 @PactBroker()
-class VerifyPactsRunSetsApiController {
+class VerifyPactsAllControllers {
```

Comment:
I couldn't find a way to effectively separate these tests into one class per controller - and I'm not sure if that would have benefitted us much anyway. Instead, this file/class has been renamed to reflect the fact that it verifies *all* controllers.